ykrt/src/trace/mod.rs

```diff
@@ -43,10 +43,10 @@ pub fn default_tracer() -> Result<Arc<dyn Tracer>, Box<dyn Error>> {
 /// Represents a thread which is currently tracing.
 pub trait ThreadTracer {
     /// Stop collecting a trace of the current thread.
-    fn stop_collector(self: Box<Self>) -> Result<Box<dyn UnmappedTrace>, InvalidTraceError>;
+    fn stop_collector(self: Box<Self>) -> Result<Box<dyn RawTrace>, InvalidTraceError>;
 }
 
-pub trait UnmappedTrace: Send {
+pub trait RawTrace: Send {
```

Comment:
Let's add a good docstring while this is fresh in our mind?

Reply:
Hah, next commit :)

Reply:
Great minds, great minds :)